Source/Scene/Model.js

```diff
@@ -1543,6 +1682,10 @@ define([
             }
         }
 
+        if (usesExtension(model, 'WEB3D_quantized_attributes')) {
+            vs = modifyShaderForQuantizedAttributes(vs, id, model, context);
```

Comment:
I'm pretty sure we would just have to go over the meshes once, not once per shader.